java/dagger/Module.java

```diff
@@ -32,7 +32,7 @@
    * Additional {@code @Module}-annotated classes from which this module is
    * composed. The de-duplicated contributions of the modules in
    * {@code includes}, and of their inclusions recursively, are all contributed
-   * to the object graph.
+   * to the component.
```

Comment:
https://github.com/google/dagger/search?utf8=%E2%9C%93&q=%22object+graph%22&type=

I'm not sure what the wording should be. I'm abandoning this, but feel free to fix those if you'd like.